tests/test_chunk.py

```diff
@@ -120,8 +121,9 @@ def test_indexing(self, extents, filtered, unfiltered):
     def test_pointcount(self, filtered, unfiltered, test_point_count, shatter_config, storage):
 
         with storage.open('w') as tdb:
-            fc = run(filtered, shatter_config, storage, tdb)
-            ufc = run(unfiltered, shatter_config, storage, tdb)
+            start_time = datetime.datetime.now().timestamp() * 1000
+            fc = run(filtered, shatter_config, storage, tdb, start_time)
```

Comment:
why thread `start_time` through all these methods instead of store it on the shatter_config?